### DISCUSSION TITLE
How to run with minute data

### DISCUSSION THREAD

**Discussion by saksinha1** [AI4Finance-Foundation/FinRL#1015] [Q&A]
How can I run FinRL using minute dataset?

**Comment by YangletLiu** [comment]
> How can I run FinRL using minute dataset?

It should be similar as long as you are talking about backtesting, right?   You get a time series.

**Reply by saksinha1** [reply]
No, it does not work because the program takes only date-wise dataset. If I try to run using minute-wise dataset, it generates error. I would appreciate if you could guide me where all changes should be made to run minute-wise datasets.

**Reply by YangletLiu** [reply]
In the Podracer project, we have played with minute-level data. It May provide you some reference: https://github.com/AI4Finance-Foundation/FinRL_Podracer
